.env.example

```
@@ -1,5 +1,6 @@
-NEXT_PUBLIC_BASE_API_URL=https://api.example.com
-NEXT_PUBLIC_API_TOKEN=replace-with-prod-token
-STORYBOOK_TOKEN=replace-with-chromatic-token
+NEXT_PUBLIC_BASE_API_URL=
+NEXT_PUBLIC_WS_BASE_URL=
+NEXT_PUBLIC_API_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_API_TOKEN key should go before the NEXT_PUBLIC_BASE_API_URL key

(UnorderedKey)